src/components/login-modal/login-modal.interface.ts

```diff
@@ -2,3 +2,13 @@ export interface ILoginModalProps {
   open: boolean;
   onClose: () => void;
 }
+
+export const defaultForm = {
+  email: "",
+  password: "",
+};
+
+export type formType = {
```

Comment:
🦆 types always `FormType`

Reply:
🥔 I've updated the type name accordingly ✍️